src/test/java/scribbles/ScribblesTest.java

```diff
@@ -0,0 +1,49 @@
+package scribbles;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ScribblesTest {
+
+    @Test
+    public void parseInputTest() {
```

Comment:
Good use of JUnit to implement test cases! Comprehensive testing of software 👍

Reply:
Thank you :)